### PR TITLE
Change to use microservices

### PR DESCRIPTION
**Pull request by gwynndp**
## Description

COMPLETED
- update api urls for requests from Captures, Verify, CaptureDetail, and Tags
- update related tests
- update filters for requests
- update field names and labels in filters

IN PROGRESS
- update related tests

VERIFY QUESTIONS (approve/reject/undo captures)
- mostly working but gets foreign key constraint errors because 
 -- the grower_account_id doesn't match anything when adding the approved to the capture table
 -- the capture_id doesn't match anything when adding the tag_id/capture_id to the capture_tags table
https://www.loom.com/share/8bab35ed927b41168589ba5e04f2c042

- where should tags for raw-captures be saved?
 -- currently using the capture_tags table for queries
 -- tag : when rejecting a raw-capture there is no place to save tags... remove the ability to add custom tags from UX in this case?

- what fields should we keep?
 -- morphology : exists on captures table, not on raw-captures table
 -- age : exists on captures table, not on raw-captures table [decided not to collect this info anymore - remove field?]
 -- capture approval tag : no place to save this, should it become a regular tag?

UP NEXT

- move grower detail to microservice
- fix report cards

DEPENDS ON API UPDATES
- https://github.com/Greenstand/treetracker-query-api/pull/122
- https://github.com/Greenstand/treetracker-field-data/pull/58
- https://github.com/Greenstand/treetracker-field-data/pull/57
- https://github.com/Greenstand/treetracker-api/pull/92

**Issue(s) addressed**

- Resolves #issue_number

**What kind of change(s) does this PR introduce?**

- [x] Enhancement
- [ ] Bug fix
- [x] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: src/components/Captures/CaptureTable.js [hunk 1]
```diff
@@ -355,16 +362,22 @@ export const formatCell = (
...
-      capture.rejectionReason,
+      capture.age !== null && capture.age >= 0 ? `age: ${capture.age}` : '',
+      capture.morphology ? `morphology: ${capture.morphology}` : '',
+      capture.captureApprovalTag
+        ? `approval tag: ${capture.captureApprovalTag}`
+        : '',
+      capture.rejectionReason
+        ? `rejection tag: ${capture.rejectionReason}`
+        : '',
       ...additionalTags,
```
Comment (gwynndp) [inline src/components/Captures/CaptureTable.js:377]: This fixes the existing display of 'tags' that wasn't working

File: .env.development [hunk 2]
```diff
@@ -1,13 +1,14 @@
 REACT_APP_WEBMAP_DOMAIN=http://dev.treetracker.org
 REACT_APP_API_ROOT=https://dev-k8s.treetracker.org/api/admin
 REACT_APP_EARNINGS_ROOT=https://dev-k8s.treetracker.org/earnings
+REACT_APP_FIELD_DATA_ROOT=http://localhost:3003
```
Comment (nmcharlton) [inline .env.development:4]: Try not to include `localhost` APIs in .env.development. Instead, put them in `.env.local`

File: .env.production [hunk 3]
```diff
@@ -1,6 +1,7 @@
 REACT_APP_WEBMAP_DOMAIN=https://treetracker.org
 REACT_APP_API_ROOT=https://prod-k8s.treetracker.org/api/admin
 REACT_APP_EARNINGS_ROOT=https://prod-k8s.treetracker.org/earnings
+REACT_APP_FIELD_DATA_ROOT=https://dev-k8s.treetracker.org/field-data
```
Comment (nmcharlton) [inline .env.production:4]: Should be `prod-k8s`

File: .env.staging [hunk 4]
```diff
@@ -1,6 +1,7 @@
 REACT_APP_WEBMAP_DOMAIN=http://staging.treetracker.org
 REACT_APP_API_ROOT=https://prod-k8s.treetracker.org/api/admin
 REACT_APP_EARNINGS_ROOT=https://prod-k8s.treetracker.org/earnings
+REACT_APP_FIELD_DATA_ROOT=https://dev-k8s.treetracker.org/field-data
```
Comment (nmcharlton) [inline .env.staging:4]: Should be `prod-k8s`

File: .env.test [hunk 5]
```diff
@@ -1,6 +1,7 @@
 REACT_APP_WEBMAP_DOMAIN=http://test.treetracker.org
 REACT_APP_API_ROOT=https://test-k8s.treetracker.org/api/admin
 REACT_APP_EARNINGS_ROOT=https://test-k8s.treetracker.org/earnings
+REACT_APP_FIELD_DATA_ROOT=https://dev-k8s.treetracker.org/field-data
```
Comment (nmcharlton) [inline .env.test:4]: `test-k8s`

File: src/components/DashStat.container.js [hunk 6]
```diff
@@ -21,7 +22,7 @@ function DashStatTotalCaptures(props) {
   const [total, setTotal] = useState(null);
 
   const getTotal = async () => {
-    console.log('-- dash getTotal');
+    log.debug('-- dash getTotal');
     const { count } = await api.getCaptureCount(activeFilter);
```
Comment (nmcharlton) [inline src/components/DashStat.container.js:26]: The standard dashboard cards no longer work.
They will soon be replaced by the new Org cards, but we'll need to keep them live for the moment.

**Comment by nmcharlton** [comment]
It seems we need to move the Species schema and API forward pretty soon.

In the meantime, do you think it's worth retrofitting UUIDs to the legacy `tree_species` table?
I guess that will make data migration easier as it creates a forward to the new schema.

**Comment by nmcharlton** [comment]
@gwynndp I've been playing around with the UI and it's coming along really nicely 👍 

One thing it does highlight is the need to rethink the use of visually prominent IDs.
We should avoid UUIDs in titles, instead relegating them to the _details_ section.

**Comment by nmcharlton** [comment]
Resolves #661

**Comment by gwynndp** [comment]
> It seems we need to move the Species schema and API forward pretty soon.
> 
> In the meantime, do you think it's worth retrofitting UUIDs to the legacy `tree_species` table? I guess that will make data migration easier as it creates a forward to the new schema.

I think that will be the easiest fix for the meantime.  Then we aren't as rushed for the Species api dev.

**Pull request merged**